data/core.yaml

```diff
@@ -1907,9 +1907,9 @@ en:
         message: "{feature} has incomplete tags"
         reference: "Some features should have additional tags."
       noncanonical_brand:
-        message: "{feature} looks like a common feature with nonstandard tags"
-        message_incomplete: "{feature} looks like a common feature with incomplete tags"
-        reference: "Some features, for example retail chains or post offices, are expected to have certain tags in common."
+        message: "{feature} may be matching one of known brands"
+        message_incomplete: "{feature} may be matching one of known brands"
```

Comment:
These two strings exist to distinguish two cases:

* The suggestion changes or deletes a tag, so it needs more scrutiny
* The suggestion is purely additive, so it’s presumably less risky

If we no longer need this distinction, we can simplify the following code:

https://github.com/openstreetmap/iD/blob/5c721808c61e39d488a2a44ffb4a0f29b9b32c8d/modules/validations/outdated_tags.js#L159-L161

Then again, maybe we do still need to make some suggestions look riskier than others?

Reply:
> Now that we have #11020, it might be worth considering if these "warnings" should be downgraded to "suggestions". I've also seen many people [waste their time](https://osm.org/changeset/160937101) adding tags like `operator:type=government` to every park, because the changeset page made it seem like a serious problem that they need to fix, even if they didn't edit that feature.

Yes, I think the `message_incomplete` case might be a good candidate for this.

Reply:
On second thought, the `message_incomplete` case may be less critical, but the `message` case is the one we’re less confident about. Giving `message` a higher severity level than `message_incomplete` could send the wrong message about our confidence in the warning.

Reply:
> If we no longer need this distinction, we can simplify the following code:

I would be fine with such change, I would be fine with distinct messages of people have a good idea.

Reply:
"may be matching a known brand with nonstandard tags" sounds like brand has nonstandard tags